playbooks/roles/appsembler_reporting/tasks/main.yml

```diff
@@ -38,3 +38,15 @@
     minute: "{{ item.cron_m }}" # Minute of the hour
     job: "bash -e {{ appsembler_reporting_scripts_dir }}/{{ item.name }}_report.sh"
   with_items: "{{ appsembler_reporting_jobs }}"
+
```

Comment:
@thraxil Is logrotate generally included in Linux distros, or something that needs to be added via package manager?

Reply:
@johnbaldwin logrotate is installed on all servers from by `common_vars` role: https://github.com/appsembler/configuration/blob/appsembler/hawthorn/master/playbooks/roles/common_vars/defaults/main.yml#L117